scripts/docs/to-integrations-repo_v2

```diff
@@ -0,0 +1,186 @@
+#!/usr/bin/env python3
```

Comment:
Just replace the old script whenever you are done with these changes.

Reply:
yep, that's the plan. wanted to make it easier to review.